### ISSUE TITLE
TLC: Integrate Supplemental Instruction (SI)

### ISSUE THREAD

**Issue by a6wu** [UCSD/campus-mobile#424]
Complete the integration of the Fall 2018 project for TLC SI.

**Comment by c3bryant** [comment]
Using logic from the `fall2018` branch, add TLC SI information to applicable classes on the Classes card and list views.

@woznaut sending your way to review/provide UI

**Comment by g9singh** [comment]
first attempt at v1 with minimal testing and use of mock data completed in 7edaa5a0af29d47b68cf99858092b577e551cc5f

![oneOpen.png](https://images.zenhubusercontent.com/5b7c4f70c76bd14ae4d85c98/20e5d573-153d-4088-8684-cefafd8f2476)
![twoOpen.png](https://images.zenhubusercontent.com/5b7c4f70c76bd14ae4d85c98/c3b00cba-9c2b-4d3e-8c89-1802720550fa)
![closed.png](https://images.zenhubusercontent.com/5b7c4f70c76bd14ae4d85c98/1c5cfdfb-b40f-4843-9b1f-71f78ae7d7fb)

**Comment by g9singh** [comment]
complete with mock data : 046db92814debe921fc29543d7bcfd13137e8f28
Most UI and data handling on client side are complete. 
Now waiting for real data to make necessary changes in data parsing.

**Comment by c3bryant** [comment]
Verified in QA. The remainder of this task is blocked by #451.

**Comment by g9singh** [comment]
completed with endpoint that has a mixture of real data and mock data in commit: 8fa1af93cbe0d50d5dfe0c7185201853cc0d7089

**Comment by c3bryant** [comment]
Verified on branch `feature/tlc-integration`, merged into 6.5

**Comment by c3bryant** [comment]
Resolved an issue where the TLC SI instructor names were not an exact match with SIS - 17 out of 35 records were affected. These records have been fixed, tested, and are ready to deploy.

**Comment by c3bryant** [comment]
TLC SI instructor names fixed, deployed, & verified
